roads.mss

```diff
@@ -38,35 +38,41 @@ as well. */
 
 #roads_med[zoom >= 9] {
   [type='motorway'],
-  [type='motorway_link'] {
-    line-color: @motorway-trunk-line;
-    [bicycle='yes'] {
-      line-color: @motorway-trunk-cycle-fill;
-    }
-  }
+  [type='motorway_link'],
   [type='trunk'],
   [type='trunk_link'] {
-      line-color: @motorway-trunk-cycle-fill;
-      [bicycle='no'] {
-        line-color: @motorway-trunk-fill;
-      }
+      line-color: @motorway-trunk-fill;
   }
-  [type='primary'] { line-color: @primary-line; }
-  [type='secondary'] { line-color: @secondary-line; }
-  [type='tertiary'] { line-color: @standard-line; }
+  [type='primary'] {
```

Comment:
We should probably drop `primary-line`, `secondary-line` and `standard-line` from road colors, they are unused after this change.